deploy/gk-deploy

```diff
@@ -468,20 +479,22 @@ if [[ ${LOAD} -eq 0 ]]; then
 fi
 
 if [[ $GLUSTER -eq 1 ]] && [[ ${LOAD} -eq 0 ]]; then
-
+  if [[ -z ${GLUSTER_LABEL} ]]; then
+    GLUSTER_LABEL=glusterfs
+  fi
   while read -r node; do
     debug "Marking '${node}' as a GlusterFS node."
-    eval_output "${CLI} label nodes ${node} storagenode=glusterfs 2>&1"
+    eval_output "${CLI} label nodes ${node} storagenode=${GLUSTER_LABEL} 2>&1"
     if [[ ${?} -ne 0 ]]; then
       output "Failed to label node '${node}'"
       abort
     fi
   done <<< "$(echo -e "${NODES}")"
   debug "Deploying GlusterFS pods."
   if [[ "${CLI}" == *oc\ * ]]; then
-    eval_output "${CLI} process glusterfs | ${CLI} create -f - 2>&1"
+    eval_output "${CLI} process ${OC_PROCESS_VAL_SWITCH} NODE_LABEL=${GLUSTER_LABEL} glusterfs | ${CLI} create -f - 2>&1"
   else
-    eval_output "${CLI} create -f ${TEMPLATES}/glusterfs-daemonset.yaml 2>&1"
+    eval_output "sed -e 's/storagenode\: glusterfs/storagenode\: '"${GLUSTER_LABEL}"'/g' ${TEMPLATES}/glusterfs-daemonset.yaml | ${CLI} create -f - 2>&1"
```

Comment:
taken care of. @obnoxxx ++ for the tests. Feels more confident about the change.